src/components/manifold-subscription-list/manifold-subscription-list.tsx

```diff
@@ -0,0 +1,88 @@
+import { Component, Element, Prop, h, Watch } from '@stencil/core';
+import { Connection } from '@manifoldco/manifold-init-types/types/v0';
+import { GraphqlError } from '@manifoldco/manifold-init-types/types/v0/graphqlFetch';
+import { SubscriptionsQuery, SubscriptionsQueryVariables } from '../../types/graphql';
+import ListCard from './components/ListCard';
+import query from './subscriptions.graphql';
+
+@Component({
+  tag: 'manifold-subscription-list',
+})
+export class ManifoldSubscriptionList {
+  @Element() el: HTMLElement;
+
+  @Prop({ mutable: true }) connection?: Connection;
+  @Prop({ mutable: true }) loading?: boolean = false;
+  @Prop({ mutable: true }) errors?: GraphqlError[];
+  @Prop({ mutable: true }) data?: SubscriptionsQuery;
+
+  /**
+   * Component heading text
+   */
+  @Prop() heading?: string;
+  /**
+   * Owner ID for subscriptions
+   */
+  @Prop() owner?: string;
+
+  @Watch('owner') async getSubscriptions(owner?: string) {
+    if (!owner) {
```

Comment:
Ah, just realized something here actually.

https://stenciljs.com/docs/reactive-data#watch-decorator

When you use `Watch`, this function will be invoked when the value of `this.owner` changes, and the arguments to the function will be the `oldValue` and the `newValue`. So I think the way that you have it now may not behave consistently between how you've called it from the constructor and how it will be called as a result of a property change.

Reply:
those docs show `watchHandler(newValue: boolean, oldValue: boolean)` I think this is what we want